apps/spruce/src/constants/task.ts

```diff
@@ -283,3 +283,5 @@ export const finishedTaskStatuses = [
 // Task name and build variant for the commit queue. Both are owned by Evergreen.
 export const mergeTaskName = "merge-patch";
 export const mergeTaskVariant = "commit-queue-merge";
+
+export const largeNumFinalizedTasksThreshold = 1000;
```

Comment:
If you do move the banner into one component I'd perhaps colocate this value since it seems unlikely to be used elsewhere